build.gradle

```diff
@@ -136,7 +136,7 @@ ext {
     androidxWorkVersion = "2.4.0"
 
     daggerVersion = '2.29.1'
-    fluxCVersion = '1.9.0-beta-8'
+    fluxCVersion = '14ddf035a43593b9185cda8e2abcc151d4ffd724'
```

Comment:
This value should be updated with the version from https://github.com/wordpress-mobile/WordPress-FluxC-Android/releases once a new release is created based on the merge commit of the FluxC PR.